CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#### 0.12.0+4
+  * Support package names with hyphens (even though it violates dart style).
```

Comment:
I thought this was a pub requirement, not just a style suggestion.

My understanding is that the hope was for a future where imports didn't use strings, but could use identifiers, and in that world, you would need this restriction